.github/actions/setup-and-build/action.yml

```diff
@@ -34,19 +29,8 @@ runs:
       id: run
       shell: bash
       run: |
-        if [ "${{ inputs.use-python-luthier }}" = "true" ]; then
-          echo "runtime=python" >> "$GITHUB_OUTPUT"
-          echo "ext=py" >> "$GITHUB_OUTPUT"
-        else
           echo "runtime=lute" >> "$GITHUB_OUTPUT"
           echo "ext=luau" >> "$GITHUB_OUTPUT"
```

Comment:
could just use lute/luau inline then instead of using variables

Reply:
good call, fixed that